samples/secure_tunneling/secure_tunnel/main.cpp

```diff
@@ -78,6 +78,16 @@ int main(int argc, char *argv[])
     string proxyUserName;
     string proxyPassword;
 
+    /*
+     * For internal testing
+     */
+    bool isTest = false;
+    int testCount = 5;
+    if (s_cmdOptionExists(argv, argv + argc, "--test"))
```

Comment:
bool isTest = s_cmdOptionExists(argv, argv + argc, "--test");

maybe

Reply:
This is so much cleaner. Thanks!